### PR TITLE
Update Mothur 1.48.0

### PR DESCRIPTION
**Pull request by gallardoalba**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [ ] - This PR updates an existing tool or tool collection
* [x] - This PR does something else (explain below)

This PR modifies some parameter names in order to match the original ones and include missing parameters.

**Modified wrappers**

* [x] align.check.xml
* [x] align.seqs.xml
- Rename parameters
* [ ] amova.xml
* [ ] anosim.xml
* [ ] bin.seqs.xml
* [x] biom.info.xml
* [x] chimera.bellerophon.xml
- Add new parameters
* [x] chimera.ccode.xml
- Add new parameters
* [x] chimera.check.xml
* [ ] chimera.perseus.xml
- Add new parameters
* [x] chimera.pintail.xml
- Add new parameters
* [x] chimera.slayer.xml
- Add new parameters
* [x] chimera.uchime.xml
- Add new parameters
* [x] chimera.vsearch.xml
- Add new parameters
* [x] chop.seqs.xml
* [x] classify.otu.xml
* [x] classify.seqs.xml
* [ ] classify.tree.xml
* [x] clearcut.xml
* [ ] cluster.classic.xml
* [x] cluster.fragments.xml
* [ ] cluster.split.xml
* [x] cluster.xml
* [ ] collect.shared.xml
* [x] collect.single.xml
* [x] consensus.seqs.xml
* [ ] cooccurrence.xml
* [ ] corr.axes.xml
* [x] count.groups.xml
* [x] count.seqs.xml
* [ ] create.database.xml
* [ ] degap.seqs.xml
* [ ] deunique.seqs.xml
* [ ] deunique.tree.xml
* [x] dist.seqs.xml
- Add new parameters
* [x] dist.shared.xml
- Add new parameters
* [x] fastq.info.xml
* [ ] filter.seqs.xml
* [x] filter.shared.xml
- Add new parameters
* [x] get.communitytype.xml
- Add new parameters
* [x] get.coremicrobiome.xml
* [ ] get.dists.xml
* [ ] get.group.xml
* [x] get.groups.xml
* [ ] get.label.xml
* [x] get.lineage.xml
* [x] get.mimarkspackage.xml
* [ ] get.otulabels.xml
* [ ] get.otulist.xml
* [x] get.oturep.xml
- Add new parameters
* [ ] get.otus.xml
* [ ] get.rabund.xml
* [ ] get.relabund.xml
* [ ] get.sabund.xml
* [x] get.seqs.xml
- Add new parameteres
* [x] get.sharedseqs.xml
* [x] heatmap.bin.xml
* [x] heatmap.sim.xml
* [ ] homova.xml
* [x] indicator.xml
* [x] lefse.xml
- Add new parameters
* [x] libshuff.xml
* [x] list.otulabels.xml
* [x] list.seqs.xml
- Add new parameters
* [ ] macros.xml
* [x] make.biom.xml
* [x] make.contigs.xml
- Add new parameters
* [ ] make.design.xml
* [ ] make.fastq.xml
* [ ] make.group.xml
* [x] make.lefse.xml
* [x] make.lookup.xml
* [x] make.shared.xml
- Rename parameters
-  Add new parameters
* [ ] make.sra.xml
* [x] mantel.xml
* [ ] merge.count.xml
* [ ] merge.files.xml
* [x] merge.groups.xml
* [x] merge.sfffiles.xml
* [ ] merge.taxsummary.xml
* [x] metastats.xml
- Add new parameters
* [x] mimarks.attributes.xml
* [x] nmds.xml
* [x] normalize.shared.xml
* [ ] otu.association.xml
* [x] otu.hierarchy.xml
- Add new parameters
* [x] pairwise.seqs.xml
- Add new parameters
* [ ] parse.list.xml
* [ ] parsimony.xml
* [ ] pca.xml
* [ ] pcoa.xml
* [x] pcr.seqs.xml
- Add new parameters
* [x] phylo.diversity.xml
* [ ] phylotype.xml
* [x] pre.cluster.xml
* [ ] primer.design.xml
* [x] rarefaction.shared.xml
- Add new parameters
* [x] rarefaction.single.xml
- Add new parameters
* [ ] remove.dists.xml
* [x] remove.groups.xml
* [x] remove.lineage.xml
* [ ] remove.otulabels.xml
* [ ] remove.otus.xml
* [ ] remove.rare.xml
* [x] remove.seqs.xml
- Add new parameters
* [x] rename.seqs.xml
* [ ] reverse.seqs.xml
* [x] screen.seqs.xml
* [ ] sens.spec.xml
* [x] seq.error.xml
* [x] sffinfo.xml
* [ ] shhh.flows.xml
* [ ] shhh.seqs.xml
* [ ] sort.seqs.xml
* [ ] split.abund.xml
* [ ] split.groups.xml
* [x] sub.sample.xml
- Add new parameters
* [ ] summary.qual.xml
* [x] summary.seqs.xml
- Add new parameters
* [x] summary.shared.xml
* [x] summary.single.xml
* [ ] summary.tax.xml
* [ ] taxonomy-to-krona.xml
* [ ] tool_data_table_conf.xml.sample
* [x] tree.shared.xml
* [x] trim.flows.xml
* [ ] trim.seqs.xml
* [ ] unifrac.unweighted.xml
* [ ] unifrac.weighted.xml
* [ ] unique.seqs.xml
* [ ] venn.xml

### CODE REVIEW

File: tools/mothur/biom.info.xml [hunk 1]
```diff
@@ -27,6 +28,10 @@ echo 'biom.info(
     ]]></command>
     <inputs>
         <param argument="biom" type="data" format="biom1" label="Biom input dataset"/>
+        <param name="format" type="select" label="Type of Biom file">
```
Comment (bernt-matthias) [inline tools/mothur/biom.info.xml:31]: Guess this switches between the `biom1` datatype and the `biom2` (which is hdf5 if I remember correctly).

So we could try to set `format` in the command section based on the input's ext and the format of the out.

Regarding the python script comparing the c++ sources and the Galaxy tool one could check if `"PARAMNAME="` appears in the command.

File: tools/mothur/align.check.xml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<tool profile="16.07" id="mothur_align_check" name="Align.check" version="@WRAPPER_VERSION@.0">
+<tool profile="16.07" id="mothur_align_check" name="Align.check" version="@WRAPPER_VERSION@.1">
```
Comment (bgruening) [inline tools/mothur/align.check.xml:1]: You should bump all those profiles and probably template them out in the macro.

**Comment by bernt-matthias** [comment]
:( 

Hey @gallardoalba would it be possible to share the python code that you used for updating all this?

**Comment by gallardoalba** [comment]
> :(
> 
> Hey @gallardoalba would it be possible to share the python code that you used for updating all this?

Sure!

**Pull request closed without merging**